### ISSUE TITLE
Drop Q2 everywhere

### ISSUE THREAD

**Issue by AleCandido** [NNPDF/pineko#54]
We should update code and docs not to use $Q^2$ in place of $\mu_F^2$, in order to avoid ambiguity, and resulting confusion and bugs.

- [ ] fix docs
- [ ] fix code

**Comment by felixhekhorn** [comment]
be aware that there is still a slight ambiguity: from PineAPPL you get a `mur2_grid` and then when calling alpha_s you still need to do `[as(xir*xir*mur2) for mur2 in mur2_grid]`, because PineAPPL really means the "default renormalization scale" (@cschwan correct me if I'm wrong). So if you want to go fully explicit one should call id `default_mur2_grid` or similar

**Comment by cschwan** [comment]
> be aware that there is still a slight ambiguity: from PineAPPL you get a `mur2_grid` and then when calling alpha_s you still need to do `[as(xir*xir*mur2) for mur2 in mur2_grid]`, because PineAPPL really means the "default renormalization scale" (@cschwan correct me if I'm wrong).

Correct.

> So if you want to go fully explicit one should call id `default_mur2_grid` or similar

This difference is usually understood. If you really want to be explicit about it I'd suggest 'central'; the scale variations vary both scales around the central choice.

**Comment by AleCandido** [comment]
> If you really want to be explicit

I don't believe we need to be so explicit here. The grid we store is the non-varied one, and multiplying `xir` tells you once more.

Something we want to do, instead, is to perform consistently the replacement, and document it: we should write `muf` for fact and `mur` for ren, and specify for each function/method/attribute returning them what is returning, possibly remarking it is the central scale.
